topics/statistics/tutorials/iwtomics/data-library.yaml

```diff
@@ -1,6 +1,17 @@
 ---
-libraries:
+destination:
+  name: GTN - Material
+  type: library
+  description: Galaxy Training Network Material
+  synopsis: Galaxy Training Network Material. See https://training.galaxyproject.org
+items:
+- name: Statistics and machine learning
+  description: Statistical Analyses for omics data and machine learning using Galaxy
+    tools
+  items:
   - name: ETn_example
-    files:
-      - url: https://doi.org/10.5281/zenodo.1184682
-        file_type: "zip"
+    items:
+    - url: https://doi.org/10.5281/zenodo.1184682
```

Comment:
Mh, we need to reupload this and use single files I think. This is not your problem but maybe you can create an issue?